(Dashboards) Add dual y-axis ability for Timeseries widget (#36026)
DataDog/documentation: content/en/dashboards/widgets/timeseries.md

### Markers

To add markers for additional datasets, click {{< ui >}}Add Marker{{< /ui >}} in the {{< ui >}}Markers{{< /ui >}} section.

1. Select a Line or Range and input a value or a range or values.
2. In the {{< ui >}}Show as{{< /ui >}} field, select an alerting status/color and choose from a solid, bold, or dashed horizontal line.
3. To add a label that displays on the bottom left of the timeseries widget, define a value for the y-axis and click the {{< ui >}}Label{{< /ui >}} checkbox.

### Dual y-axis

> To display two y-axes on the same graph, click **Add Graph**. Use the **Y-axis** dropdown to select whether each y-axis is displayed on the left or right.

Reviewer: One clarification, a graph can have more than 2 requests, you could have multiple requests assigned to the same axis, you can assign as many requests as you want to each side. I would rephrase this to make it clearer that the thing being assigned to a side is a request, not a y axis. 

> Use the **Y-axis** dropdown to select whether this request's data associated with the left or right axis

Author: thank you! i updated with your suggestion and added, "you can assign as many requests as you want to each side"

> To display more than one y-axis on the same graph, click **Add Graph**. Use the **Y-axis** dropdown to select whether the request's data will be associated with the left or right axis. You can assign as many requests as you want to each side.

Reviewer 2: ```suggestion
To display more than one y-axis on the same graph, click **Add Graph**. Use the **Y-axis** dropdown to associate the request's data with the left or right axis. You can assign any number of requests to each side.
```

Author: such a good suggestion @brett0000FF. thank you!

> {{< img src="dashboards/widgets/timeseries/dual_y_axis.png" alt="The Timeseries widget shows two metric queries: system.uptime in purple on the left y-axis and system.disk.used in orange on the right y-axis." style="width:100%;" >}}

Reviewer: Nice annotations and alt text 👍

Author: thank you :)

| `Max`                 | No       | Specify the maximum value to show on the y-axis. It takes a number or `Auto` as the default value.                                                                                                                        |
| `Scale`               | No       | Specifies the scale type. Possible values include:<br>- *linear*: A linear scale (default).<br>- *log*: A logarithmic scale.<br>- *pow*: A Power of 2 scale (2 is default, modify in JSON).<br>- *sqrt*: A square root scale. |
| `Always include zero` | No       | Always include zero or fit the y-axis to the data range. The default is to always include zero.                                                                                                                             |

> <div class="alert alert-info">Because the mathematical log function does not accept negative values, the Datadog log scale only works if values are of the same sign (everything > 0 or everything < 0). Otherwise, Datadog returns an empty graph.</div>

Reviewer: This change seems unrelated to the Y axis, if intentional should we mention it in the PR description?

Author: thanks! yeah i'll add that right now
